File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int f=0;
        for(int i=0; i<nums.size(); i++){
            if(nums[i]!=val){
                nums[f]=nums[i];
                f++;
            }
        }
        return f;
    }
};

File: 0027-remove-element/0027-remove-element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
        let i;
        let s=0;
        let n=nums.length;

        if(n==0)
        return 0;
        for(i=0; i<n; i++)
        {
            if(nums[i]!=val)
            {
                nums[s]=nums[i];
                s++;
            }
        }
        return s;
    }
